components/admin_console/schema_admin_settings.jsx

```diff
@@ -890,10 +902,6 @@ export default class SchemaAdminSettings extends React.Component {
             this.setState(getStateFromConfig(config));
         }
 
-        if (callback) {
```

Comment:
Whats the reason for removing the callback?

Reply:
it was not being used anywhere so I decided to get rid of it